src/NanoManager.cpp

```diff
@@ -85,28 +85,29 @@ void NanoManager::nanoSummonHandler(CNSocket* sock, CNPacketData* data) {
     if (pkt->iNanoSlotNum > 2)
         return;
 
-    int nanoId = plr.plr.equippedNanos[pkt->iNanoSlotNum];
+    int nanoId = plr.equippedNanos[pkt->iNanoSlotNum];
 
     if (nanoId > 36)
         return; // sanity check
 
-    sNano nano = plr.plr.Nanos[nanoId];
+    sNano nano = plr.Nanos[nanoId];
 
     // Send to other players
-    INITSTRUCT(sP_FE2CL_NANO_ACTIVE, pkt1);
+    for (CNSocket* s : PlayerManager::players[sock].viewable) {
```

Comment:
You seem to have accidentally reverted 756074cc62b49907086fb2f515d6343813bc106f. I agree with the rest of your changes in here, just merge the two.

Reply:
Ya things got messed up. Should be as said commit.

Reply:
Don't forget to pull the assignments to the members of `pkt1` out of the loop as well. That commit also fixes some indentation errors that I had made and then fixed, but they remain unfixed in this patch.